contact.html

```diff
@@ -39,14 +40,14 @@
                 <h2>Inquiries</h2>
                 <div class="col-md-6">
                     <h4>General</h4>
-                    <p>Contact Jatin Patil (Project Director) at:
-                        <a href="mailto:jjpatil@uwaterloo.ca">jjpatil@uwaterloo.ca</a>
+                    <p>Contact James Oswald (Managing Director) at:
+                        <a href="mailto:jtoswald@uwaterloo.ca">jtoswald@uwaterloo.ca</a>
                     </p>
                 </div>
                 <div class="col-md-6">
                     <h4>Sponsorship</h4>
-                    <p>Contact Vyshakh Sanjeev (Business Development Lead) at:
-                        <a href="mailto:vsanjeev@uwaterloo.ca">vsanjeev@uwaterloo.ca</a>
+                    <p>Contact Waleed Ahmed (Business Development Lead) at:
+                        <a href="mailto:@uwaterloo.ca">vsanjeev@uwaterloo.ca</a>
```

Comment:
There's nothing in the prefix of `@uwaterloo.ca`, but it still displays `vsanjeev@uwaterloo.ca`. Either remove the `a` tag or have a correct email.